migrations/V5__Add_Business_Case_Table.sql

```diff
@@ -0,0 +1,52 @@
+create table business_case (
+   id uuid PRIMARY KEY not null,
+   eua_user_id text not null,
+   project_name text,
+   requester text,
+   requester_phone_number text,
+   business_owner text,
+   business_need text,
+   cms_benefit text,
+   priority_alignment text,
+   success_indicators text,
+
+   as_is_title text,
+   as_is_summary text,
+   as_is_pros text,
+   as_is_cons text,
+   as_is_cost_savings text,
+
+   preferred_title text,
+   preferred_summary text,
+   preferred_acquisition_approach text,
+   preferred_pros text,
+   preferred_cons text,
+   preferred_cost_savings text,
+
+   alternative_a_title text,
+   alternative_a_summary text,
+   alternative_a_acquisition_approach text,
+   alternative_a_pros text,
+   alternative_a_cons text,
+   alternative_a_cost_savings text,
+
+   alternative_b_title text,
+   alternative_b_summary text,
+   alternative_b_acquisition_approach text,
+   alternative_b_pros text,
+   alternative_b_cons text,
+   alternative_b_cost_savings text
+);
+
+CREATE TYPE lifecycle_cost_phase AS ENUM ('Initiate', 'Operations & Maintenance');
+CREATE TYPE lifecycle_cost_solution AS ENUM ('As Is', 'Preferred', 'A', 'B');
+CREATE TYPE lifecycle_cost_year AS ENUM ('1', '2', '3', '4', '5');
+
+create table estimated_lifecycle_cost (
```

Comment:
Because there's the "Add phase" button, we may need something more fine-grained. If we have multiple phases in the same year, in the same lifecycle_cost_phase, we need a way to order them. Happy to chat about this

Reply:
do we need to order them? What's the reason for that?

Reply:
I'm not sure... maybe we don't